app/routes/interestgroups/components/InterestGroupList.tsx

```diff
@@ -38,32 +38,44 @@ const InterestGroupList = () => {
   return (
     <Content>
       <Helmet title="Interessegrupper" />
-      <div className={styles.section}>
-        <NavigationTab title="Interessegrupper" />
-        <p>
+      <NavigationTab title="Interessegrupper" />
+      <Flex
+        wrap
+        gap={10}
+        margin={'0 0 30px'}
+        justifyContent="space-between"
+        alignItems="center"
+      >
+        <p style={{ margin: 0 }}>
           <Link to="/pages/generelt/39-praktisk-informasjon">Her</Link> finner
           du all praktisk informasjon knyttet til våre interessegrupper.
         </p>
         {canCreate && (
-          <Link to="/interest-groups/create" className={styles.link}>
+          <Link to="/interest-groups/create">
             <Button>Lag ny interessegruppe</Button>
           </Link>
         )}
-      </div>
-      <div className="groups">
-        {activeGroups.map((g) => (
-          <InterestGroupComponent group={g} key={g.id} active={true} />
-        ))}
+      </Flex>
+
+      {activeGroups.map((g) => (
+        <InterestGroupComponent group={g} key={g.id} active={true} />
+      ))}
+
+      <div className={styles.inactiveHeader}>
         <h2>Ikke-aktive interessegrupper</h2>
         <p>
           Send gjerne e-post til
-          <a href="mailTo:interessegrupper@abakus.no"> oss </a> hvis du ønsker å
-          åpne en av disse igjen!
+          <a href="mailTo:interessegrupper@abakus.no">
+            {' '}
```

Comment:
Dunno if it makes any difference, but it feels more legal to add this space outside of the a-tag than inside 🤷